### PR TITLE
TEST: Added Berlin ZoomIn test

### PR DESCRIPTION
**Pull request by ninok**
* Add new Berlin zoom testcase that mimics HERE Maps for JavaScript testcase
* Add save functionality
* Allow configuration of themes via webpack (for internal themes)

### CODE REVIEW

File: @here/harp-examples/src/performance_benchmark.ts [hunk 1]
```diff
@@ -631,7 +648,7 @@ export namespace PerformanceBenchmark {
                 "6": 6,
                 "8": 8
             },
-            PhasedLoading: true,
+            PhasedLoading: false,
```
Comment (nzjony) [inline @here/harp-examples/src/performance_benchmark.ts]: This should have already been removed? Maybe make a ticket for this.

File: @here/harp-examples/src/performance_benchmark.ts [hunk 2]
```diff
@@ -572,6 +588,7 @@ export namespace PerformanceBenchmark {
             },
             CanvasSize: {
                 default: undefined,
+                "1100×900": "1100×900",
```
Comment (nzjony) [inline @here/harp-examples/src/performance_benchmark.ts]: Why this resolution?

File: @here/harp-examples/webpack.config.js [hunk 3]
```diff
@@ -19,6 +21,15 @@ const harpFontResourcesPath = path.dirname(require.resolve("@here/harp-fontcatal
 const isProduction = process.env.NODE_ENV === "production";
 const harpBundleSuffix = isProduction ? ".min" : "";
 
+themeList = {
```
Comment (nzjony) [inline @here/harp-examples/webpack.config.js]: why is this moved to the webpack config? It would make sense if there are multiple files that need it, but it seems a bit strange to move it away from the performance test.

Comment (ninok) [reply @here/harp-examples/webpack.config.js]: b/c the theme list is different for the internal build (i.e. it uses the internal styles).
Please see my internal change as well

File: @here/harp-examples/src/performance_benchmark.ts (hunk 2 above)
Comment (ninok) [reply @here/harp-examples/src/performance_benchmark.ts]: JSAPI is using it for their tests.

File: @here/harp-examples/src/performance_benchmark.ts (hunk 1 above)
Comment (ninok) [reply @here/harp-examples/src/performance_benchmark.ts]: true, will remove in a follow up commit
